README.md

```diff
@@ -17,10 +17,14 @@ Be sure to read [How To submit your suggestion](./how-to/submit-your-suggestion.
 
 # Table of Contents
 
-- [Introduction](#introduction)
-  - [Suggest a Resource](#suggesting-a-resource)
+- [SheSharp knowledge hub](#shesharp-knowledge-hub)
+  - [Introduction](#introduction)
+  - [Suggesting a resource](#suggesting-a-resource)
 - [Table of Contents](#table-of-contents)
-  - [Social Media](#social-media)
+- [Social Media](#social-media)
+  - [Instagram profiles](#instagram-profiles)
+  - [Linkedin profiles](#linkedin-profiles)
```

Comment:
We could add Twitter as well, a while back there was some one asking about it.
You are not obligated to include this in any way!

Slack: https://she-sharp.slack.com/archives/CPHESUZ5H/p1664192962620809

Reply:
Feel free to add that section @TessavWalstijn! :)

Reply:
I moved it to a new ticket: https://github.com/shesharpnl/knowledge-hub/issues/4